backend/src/tasks/test/wappalyzer.test.ts

```diff
@@ -12,46 +46,124 @@ jest.mock('simple-wappalyzer', () => async () => [
     icon: 'Drupal.svg',
     website: 'https://drupal.org',
     cpe: 'cpe:/a:drupal:drupal'
-  },
-  {
-    name: 'Apache',
-    slug: 'apache',
-    categories: [
-      { id: 22, slug: 'web-servers', name: 'Web servers', priority: 8 }
-    ],
-    confidence: 100,
-    version: '',
-    icon: 'Apache.svg',
-    website: 'http://apache.org',
-    cpe: 'cpe:/a:apache:http_server'
-  },
-  {
-    name: 'PHP',
-    slug: 'php',
-    categories: [
-      {
-        id: 27,
-        slug: 'programming-languages',
-        name: 'Programming languages',
-        priority: 5
-      }
-    ],
-    confidence: 100,
-    version: '',
-    icon: 'PHP.svg',
-    website: 'http://php.net',
-    cpe: 'cpe:/a:php:php'
   }
-]);
+];
+
+const commandOptions: CommandOptions = {
+  organizationId: 'organizationId',
+  organizationName: 'organizationName',
+  scanId: 'scanId',
+  scanName: 'scanName',
+  scanTaskId: 'scanTaskId'
+};
 
 describe('wappalyzer', () => {
-  test('basic test', async () => {
-    await wappalyzer({
-      organizationId: 'organizationId',
-      organizationName: 'organizationName',
-      scanId: 'scanId',
-      scanName: 'scanName',
-      scanTaskId: 'scanTaskId'
-    });
+  let testDomain: Domain;
+
+  beforeAll(() => {
+    saveDomainsToDbMock.mockResolvedValue();
+  });
+
+  beforeEach(() => {
+    testDomain = new Domain();
+    testDomain.name = 'example.com';
+    axiosMock.get.mockResolvedValue(apiResponse);
```

Comment:
Can we use `nock` to mock HTTP responses instead, for consistency with other tests?

Reply:
Sounds good, should have looked at the other tests first